src/org/etools/j1939_84/J1939_84.java

```diff
@@ -59,22 +59,13 @@ public static Logger getLogger() {
         return logger;
     }
 
-    /**
-     * Returns true if the application is being debugged
-     *
-     * @return true if it's being debugged
-     */
-    public static final boolean isDebug() {
-        return true;
-    }
-
     /**
      * Returns true if the application is under test
      *
      * @return true if it's being tested
      */
     public static final boolean isTesting() {
-        return System.getProperty(TESTING_PROPERTY_NAME, "false").equals("true");
+        return Boolean.getBoolean(TESTING_PROPERTY_NAME);
```

Comment:
This won't work.  You're converting a 'key' to a boolean.

Reply:
I think this is he appropriate method of reading boolean system properties.  See [Boolean.html#getBoolean(java.lang.String)](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/Boolean.html#getBoolean(java.lang.String)
)

Reply:
I'd be OK with 
`return System.getProperty(TESTING_PROPERTY_NAME, "false").equalsIgnoreCase("true")`

Reply:
I didn't know that method existed.